ConsoleApp1/ConsoleApp1/Program.cs

```diff
@@ -7,6 +8,8 @@ class Program
         static void Main(string[] args)
         {
             Console.WriteLine("Hello World!");
+            Console.WriteLine("Assuming develop branch..");
```

Comment:
```suggestion
            Console.WriteLine("Assuming develop branch..");
```
Change the above code - update string to "Migrating to Master"